includes/output-view.tab.inc

```diff
@@ -55,6 +55,8 @@ function islandora_chemistry_datastream_view(AbstractObject $object, $dsid = 'OB
     $mime_detect = new MimeDetect();
     $geshi = new GeSHi();
     $geshi->enable_classes();
+    // Override the default code style as it causes text-alignment issues.
```

Comment:
Should get an "XXX" saying something about browser specificity, since this doesn't appear to be an issue in Firefox.